base/testfiles/github-0591.tlg

```diff
@@ -0,0 +1,2 @@
+This is a generated file for the LaTeX2e validation system.
+Don't change this file in any respect.
```

Comment:
I think there isn't anything wrong to with checking the showlists output in case of pdftex engine as well if you start up the test (otherwise there are configs that only run unicode engines though I don't think there is one only running luatex)

Reply:
Done